duckduckgo/iOS
# Update user agent to fix breakages

Task/Issue URL: https://app.asana.com/0/361428290920652/1176477052804240
Tech Design URL: https://app.asana.com/0/361428290920652/1176477052804235

**Description**:
Updates user agent to include a safari component after our application component (similar to Chrome) and to allow site specific changes.

**Steps to test this PR**:
1. Got to `https://www.whatismybrowser.com/detect/what-is-my-user-agent` and check that the user agent reported contains our DuckDuckGo component followed by the Safari component
`Mozilla/5.0 (iPhone; CPU iPhone OS 12_4 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/15E148 DuckDuckGo/7 Safari/605.1.15`
1. From an iPhone: Visit `tdbank.com` and press "Login in" and then "Online Banking" and check that you see a login box
1. From an iPhone: Visit `svb.com`, press Login (or the little profile icon) and then "SVB Online Service" and check that you see a login box
1. From an iPad: Visit `chase.com`. Depending on screen size you may need to press the "Sign in" button to see the login box or it may already be there. Either way make sure you can get to a login box. Note: This page is really slow sometimes!
1. From a iPhone or iPad: Visit `cvs.com` (use a VPN located in the US) in both Desktop and Mobile modes and ensure it loads ok. This UA is dynamically changed and will only work for iOS >=12

<img width="584" alt="comparison1" src="https://user-images.githubusercontent.com/1652407/85287039-9d41a400-b48b-11ea-93a3-df7b88f7f06a.png">
<img width="562" alt="comparison2" src="https://user-images.githubusercontent.com/1652407/85287051-a03c9480-b48b-11ea-8c26-9e445bd9ffa9.png">



**Device Testing**:

**OS Testing**:

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
